Updating link to HCP Terraform per #710 (#713)
hashicorp/web-unified-docs: content/terraform/v1.13.x (rc)/docs/cli/cloud/index.mdx

# Use HCP Terraform with the Terraform CLI 

> The Terraform CLI integration with HCP Terraform lets you use [HCP Terraform](https://developer.hashicorp.com/terraform/cloud-docs) and [Terraform Enterprise](https://developer.hashicorp.com/terraform/enterprise) on the command line. In the documentation HCP Terraform instructions also apply to Terraform Enterprise, except where explicitly stated.

Reviewer: ```suggestion
The Terraform CLI integration with HCP Terraform lets you use [HCP Terraform](https://developer.hashicorp.com/terraform/cloud-docs) and [Terraform Enterprise](https://developer.hashicorp.com/terraform/enterprise) on the command line. The instructions for HCP Terraform also apply to Terraform Enterprise, except where explicitly stated.
```
